changedetectionio/tests/test_xpath_selector.py

```diff
@@ -277,3 +277,45 @@ def test_check_with_prefix_include_filters(client, live_server):
     assert b"Some text that will change" not in res.data #not in selector
 
     client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+
+def test_allow_server_respond_with_bytes(client, live_server):
+    res = client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+    assert b'Deleted' in res.data
+
+    # Give the endpoint time to spin up
+    time.sleep(1)
+
+    # A poorly configured non-utf-8 HTML of server-side.
+    d = b'<html lang="ko">\n<head>\n<meta http-equiv="Content-Type" content="text/html; charset=EUC-KR">\n<style>\np {\n  @charset EUC-KR;\n  color: orange;\n  }\n</style>\n</head>\n<body>\n<p>\xc8\xa5\xb5\xb7\xc0\xba \xb4\xe7\xbf\xac\xc7\xcf\xb4\xd9..</p>\n<p>If you are reading this, then server sent bytes successfully.</p>\n</body>\n</html>\n'
```

Comment:
The previous with-open can't read these bytes because it's not UTF-8. Therefore without a fix of the next commit, the test server throws an error.